pkg/cli/objectformats/objectformats.go

```diff
@@ -55,8 +55,8 @@ func GetApplicationGatewaysTableFormat() output.FormatterOptions {
 	}
 }
 
-// GetResourceTableFormat() returns a FormatterOptions struct containing two columns, one for the resource name and one for
-// the resource type.
+// GetResourceTableFormat() returns a FormatterOptions struct containing three columns, one for the resource name, one for
+// the resource type, and one for the provisioning state.
```

Comment:
There's really no need for this doc to be so specific :)

When I've encountered overly specific doc comments like this, I just remove them.